app/imports/api/methods/commentsMethods.js

```diff
@@ -0,0 +1,53 @@
+import { Meteor } from 'meteor/meteor'
+
+import { Applications } from '/imports/collections/applications.js';
+import { Comments } from '/imports/collections/comments.js';
+
+import { AccountsCommon } from 'meteor/accounts-base';
+
+export const submitComment = new ValidatedMethod({
+  name: 'comment.submit',
+
+  validate: new SimpleSchema({
+    applicationId: {
+      type: String,
+    },
+    comment: {
+      type: String,
+    },
+  }).validator(),
+
+  run({ applicationId, comment }) {
+    if (!Roles.userIsInRole(this.userId, ['committee', 'admin'])) {
```

Comment:
but this is an applaya question not maf thing. by default i dont think admin should be part of the committee process